config/explorer.php

```diff
@@ -9,9 +9,16 @@
      * https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/configuration.html
      */
     'connection' => [
-        'host' => 'localhost',
-        'port' => '9200',
-        'scheme' => 'http',
+        'host' => env('ELASTIC_HOST', 'localhost'),
+        'port' => env('ELASTIC_PORT', '9200'),
+        'scheme' => env('ELASTIC_SCHEME', 'http'),
+        // 'user' => env('ELASTIC_USER', ''),
+        // 'pass' => env('ELASTIC_PASS', ''),
+    ],
+    'api' => [
+        'cloud-id' => env('ELASTIC_CLOUD_ID', ''),
```

Comment:
This could be placed in the host I think: https://github.com/elastic/elasticsearch-php/blob/d7539aa7c414a92ee34ae95070da1bc5dd9d30c3/src/Elasticsearch/ClientBuilder.php#L458

Reply:
Yes, sorry, you're right 👍